api/agent/agent.go

```diff
@@ -1302,22 +1311,8 @@ func (c *container) UDSDockerPath() string              { return c.iofs.DockerPa
 func (c *container) UDSDockerDest() string              { return iofsDockerMountDest }
 func (c *container) DisableNet() bool                   { return c.disableNet }
 
-func (c *container) SetCallId(callId string) {
-	c.callIdLck.Lock()
-	c.callId = callId
-	c.callIdLck.Unlock()
-}
-
-func (c *container) GetCallId() string {
-	var callId string
-	c.callIdLck.Lock()
-	callId = c.callId
-	c.callIdLck.Unlock()
-	return callId
-}
-
 // WriteStat publishes each metric in the specified Stats structure as a histogram metric
-func (c *container) WriteStat(ctx context.Context, stat drivers.Stat) {
+func (c *container) WriteStat(ctx context.Context, stat driver_stats.Stat) {
```

Comment:
not blocking, just curious, what was the motivation for this? I'm inclined to move it back [later] as it conflicts with opencensus and named imports are a smell of poorly named packages, via [the guide](https://github.com/golang/go/wiki/CodeReviewComments#imports)

Reply:
Circular imports, alas - driver stats are embedded into models.Call, but the before/after stuff wants to talk about a models.Call also.

(This is probably an indication that this *does* belong on the agent.)

Reply:
ah, thanks. yea. maybe move to models or something.